go/vt/vtgate/planbuilder/operators/apply_join.go

```diff
@@ -139,11 +142,14 @@ func (aj *ApplyJoin) SetRHS(operator Operator) {
 }
 
 func (aj *ApplyJoin) MakeInner() {
-	aj.LeftJoin = false
+	if aj.IsInner() {
+		panic(vterrors.VT13001("Convert an already inner join"))
+	}
+	aj.JoinType = sqlparser.NormalJoinType
 }
```

Comment:
earlier this method was idempotent. Any reason to `panic` now on multiple calls of this method?